client/containers/Recipe.js

```diff
@@ -40,18 +49,37 @@ class Recipe extends Component {
     };
 
     if (loaded) {
+      //TODO: is there a better way that doesn't involve duplicating the whole thing?
       return (
-        <div style={recStyle} className="recipe" onLoad={() => onLoad(id)}>
-          <h1 style={h1Style}>Name: {name}</h1>
-          <div>{description ? "Yield: " + description : ""}</div>
-          <div className="ingredients" style={ingStyle}>
-            <h1 style={h1Style}>Ingredients:</h1>
-            {inventory && inventory.map(i => <Ingredient {...i} />)}
-          </div>
-          <div className="instructions">
-            <h1 style={h1Style}>Method of Prep:</h1>
-            {instructions && instructions.map(i => <Instruction {...i} />)}
-          </div>
+        <div>
+          <MediaQuery print>
```

Comment:
Might just want to use a good-ole stylesheet for this. at least for the `@media print`variant.